### ISSUE TITLE
nodejieba 2.6.0 本地 pnpm install 失败

### ISSUE THREAD

**Issue by yusixian** [Crossbell-Box/xLog#287]
尝试本地跑xLog项目，发现一直pnpm i失败，查了下nodejieba 库貌似不能2.6.0了，想问一下开发团队是如何解决的，或者有没有更详细的面向开发者的文档~
![image](https://user-images.githubusercontent.com/43498495/230107540-2bbfdd87-5040-4d98-8613-2b585efa8fe3.png)
```bash
│ node-pre-gyp info it worked if it ends with ok
│ node-pre-gyp info using node-pre-gyp@1.0.10
│ node-pre-gyp info using node@16.18.0 | win32 | x64
│ node-pre-gyp info check checked for "C:\Programming\project\xLog\node_modules\.pnpm\nodejieba@2.6.0\node_modules\nodejieb  
│ node-pre-gyp http GET https://github.com/yanyiwu/nodejieba/releases/download/v2.6.0/nodejieba-v2.6.0-node-v93-win32-x64.t  
│ node-pre-gyp ERR! install response status 404 Not Found on https://github.com/yanyiwu/nodejieba/releases/download/v2.6.0/  
│ node-pre-gyp WARN Pre-built binaries not installable for nodejieba@2.6.0 and node@16.18.0 (node-v93 ABI, unknown) (fallin  
│ node-pre-gyp WARN Hit error response status 404 Not Found on https://github.com/yanyiwu/nodejieba/releases/download/v2.6.  
│ gyp info it worked if it ends with ok
│ gyp info using node-gyp@9.3.0
│ gyp info using node@16.18.0 | win32 | x64
│ gyp info ok
```

找到了nodejieba 的 [这个 issue](https://github.com/yanyiwu/nodejieba/issues/205) 
![image](https://user-images.githubusercontent.com/43498495/230107139-8209a871-2b4a-48dc-b712-d406737f04c5.png)

**Comment by DIYgod** [comment]
macOS has not encountered this problem, perhaps consider switching to https://github.com/napi-rs/node-rs/tree/main/packages/jieba

**Comment by DIYgod** [comment]
Sorry, this is a mistake. `nodejieba` should not have been included in the dependencies and has already been fixed in 90220752a87d5d80c456eb596a43f025b9c5c3d0
